BookShelfBusinessLogic/ILibraryService.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BookShelfBusinessLogic
+{
+    public interface ILibraryService
```

Comment:
Library service became too big. Please split it. On Book,Author,Genre services and if you want you can place common logic here or split it as well. But avoid any method duplication